lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +242,57 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        conllu_sentences = []
+        mystem = Mystem()
```

Comment:
define once per pipeline instance (in initializer?)

Reply:
done